services/apps/src/apps/clients/notifications.clj

```diff
@@ -27,6 +28,14 @@
              {:content-type :json
               :body (cheshire/encode m)}))
 
+(defn- guarded-send-notification
```

Comment:
not for this PR, but presumably we should use this for more things in this namespace too